### ISSUE TITLE
Frequently getting  I/O Error

### ISSUE THREAD

**Issue by malayap47** [TakahikoKawasaki/nv-websocket-client#151]
Hi ,
   I am using 2.3 android version. But Frequently getting error after successful connection. The error is 
=An I/O error occurred while a frame was being read from the web socket: Read error: ssl=0x55a8812bb0: I/O error during system call, Connection timed out

**Comment by deepaksharma1992** [comment]
Yes please answer this question as I am getting the same error on API<21. It's working fine above API 21.

**Comment by Farhad7d7** [comment]
@TakahikoKawasaki we need some help here

**Comment by ndvbd** [comment]
Same issue here

**Comment by vlasky** [comment]
I have a helpful suggestion - try previous versions of nv-websocket-client and see if they experience the same issue.

If you find a previous version works, post the version number. This may help us determine what the breaking change was (if there was one).

I used this approach to help fix a different past bug.
